skyvern/forge/sdk/artifact/storage/test_s3_storage.py

```
@@ -447,3 +447,107 @@ async def test_content_type_guessing(
+        stored_data = obj_response["Body"].read()
+
+        # Stored data should be different from original (compressed)
+        assert stored_data != har_data
+
+        # Verify we can decompress it back to original
+        dctx = zstd.ZstdDecompressor()
+        decompressed = dctx.decompress(stored_data)
+        assert decompressed == har_data
+
+    async def test_retrieve_har_artifact_decompresses_zstd(
+        self, s3_storage: S3Storage, boto3_test_client: S3Client
```

Ruff (0.14.10)

Unused method argument: `boto3_test_client`

(ARG002)